src/code/z_actor.c

```diff
@@ -517,7 +582,26 @@ GLOBAL_ASM("asm/non_matchings/z_actor//Actor_InsertIntoTypeList.asm")
 
 GLOBAL_ASM("asm/non_matchings/z_actor//Actor_RemoveFromTypeList.asm")
 
-GLOBAL_ASM("asm/non_matchings/z_actor//Actor_FreeOverlay.asm")
+void Actor_FreeOverlay(ActorOverlayTableEntry* entry) {
+    u32 ramAddr;
+
+    if (entry->clients == 0) {
+        ramAddr = entry->ramAddr;
+        if (ramAddr != 0) {
+	    //Bit 1 - always loaded
+            if ((entry->flags & 2) == 0) {
+		//Bit 0 - don't alloc memory
+                if ((entry->flags & 1) != 0) {
+                    entry->ramAddr = 0;
+                }
+                else {
```

Comment:
Put the else on the same line as the closing brace.
```suggestion
                } else {
```